csrc/lower_validation.cpp

```diff
@@ -634,13 +673,8 @@ void fillProducerVectorizedContigRootDomains(
 
   VectorizedSetInfo& info = *it;
 
-  // info.vectorized_root_id is validated at this point to be the
-  // last concrete root domain in consumer.
-  auto consumer_root_id = info.vectorized_root_id;
-
-  auto root_id = c2p_map.at(consumer_root_id);
-
-  fillVectorizedContigRootDomains(producer_tv, contig_finder, root_id, info);
+  fillVectorizedContigRootDomains(
+      producer_tv, contig_finder, info.vectorized_producer_root_id, info);
```

Comment:
Shouldn't we check if the producer root ID is not `nullptr`?

Reply:
After the change in https://github.com/NVIDIA/Fuser/pull/298#discussion_r1187990716, I think `vectorized_producer_root_id` can no longer be `nullptr` anymore.